src/inputs/text.html

```diff
@@ -1,9 +1,15 @@
 <!--Textfield input-->
 <script type="text/ng-template" id="wgn-config-input-text">
 	<input type="text" class="form-control" name="{{ field.id }}"
-		ng-model="editing.config[field.id]"
-		ng-required="field.required"
-		placeholder="{{ field.placeholder }}"
-		ng-minlength="{{ field.minlength }}"
-		ng-maxlength="{{ field.maxlength }}">
+		   ng-model="editing.config[field.id]"
+		   ng-required="field.required"
+		   placeholder="{{ field.placeholder }}"
+		   ng-pattern="field.pattern"
```

Comment:
I believe this should be `{{ field.required }}` and `{{ field.pattern }}`?

Reply:
it actually isn't because they are not strings the field.pattern is a regex and field.required is a boolean

Reply:
the field.pattern doesn't work with {{}} took me for ever to realize that :(

Reply:
got it, thanks for clarification